scripts/install-maria.sh

```diff
@@ -1,14 +1,15 @@
 
 # Check If Maria Has Been Installed
 
+set -e
+set -x
```

Comment:
Would love if these were documented. I think all but the most hardcore scripters remember all the flags.